vocode/streaming/telephony/conversation/twilio_phone_conversation.py

```diff
@@ -140,135 +129,10 @@ async def _handle_ws_message(self, message) -> Optional[TwilioPhoneConversationW
             chunk = base64.b64decode(media["payload"])
             self.receive_audio(chunk)
         if data["event"] == "mark":
-            mark_name = data["mark"]["name"]
-            if mark_name.startswith("chunk-"):
-                utterance_id, chunk_idx = mark_name.split("-")[1:]
-                self.mark_message_queue.put_nowait(
-                    utterance_id=utterance_id,
-                    mark_message=ChunkFinishedMarkMessage(chunk_idx=int(chunk_idx)),
-                )
-            elif mark_name.startswith("utterance"):
-                utterance_id = mark_name.split("-")[1]
-                self.mark_message_queue.put_nowait(
-                    utterance_id=utterance_id,
-                    mark_message=UtteranceFinishedMarkMessage(),
-                )
+            chunk_id = data["mark"]["name"]
+            self.output_device.enqueue_mark_message(ChunkFinishedMarkMessage(chunk_id=chunk_id))
```

Comment:
there is no way to distinguish from the twilio mark message whether it came because of a `clear` message or a `media` message - that's why there is only one type of mark, to signal that Twilio has finished processing the corresponding media message.